File: C/1005.c
#include <stdio.h>
 
int main() {
 
    float a,b, media;
    scanf("%f", &a);
    scanf("%f", &b);
    media = a * 3.5;
    media = media + b * 7.5;
    media = media / 11;
    printf("MEDIA = %.5f\n", media);
}

File: Java/1005.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = ((a*3.5) + (b*7.5))/11;
        System.out.printf("MEDIA = %.5f\n", c);
    }
}
